File: tmp/clear_animals.js
const fs = require('fs');
const path = require('path');

const filePath = path.join('c:', 'Users', 'James', 'Desktop', 'cosmic-observer(3)', 'src', 'data', 'animals.ts');
let content = fs.readFileSync(filePath, 'utf8');

// Regex to match and clear species, habitat, description
content = content.replace(/("species":\s*")[^"]*(")/g, '$1$2');
content = content.replace(/("habitat":\s*")[^"]*(")/g, '$1$2');
content = content.replace(/("description":\s*")[^"]*(")/g, '$1$2');

fs.writeFileSync(filePath, content);
console.log("Successfully cleared species, habitat, and description for all animals.");

File: tmp/clear_animals.py
import re

with open(r'c:\Users\James\Desktop\cosmic-observer(3)\src\data\animals.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Clear species
content = re.sub(r'("species":\s*")[^"]*(")', r'\1\2', content)

# Clear habitat
content = re.sub(r'("habitat":\s*")[^"]*(")', r'\1\2', content)

# Clear description
content = re.sub(r'("description":\s*")[^"]*(")', r'\1\2', content)

with open(r'c:\Users\James\Desktop\cosmic-observer(3)\src\data\animals.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("Successfully cleared species, habitat, and description for all animals.")
